content/docs/conditional-rendering.md

```diff
@@ -116,13 +117,13 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
+[**Essayer sur  CodePen**](https://codepen.io/gaearon/pen/QKzAgB?editors=0010)
 
-While declaring a variable and using an `if` statement is a fine way to conditionally render a component, sometimes you might want to use a shorter syntax. There are a few ways to inline conditions in JSX, explained below.
+Déclarer une variable et utiliser une déclaration en `if` est une bonne façon de rendre un composant conditionnellement, parfois vous voudriez utiliser une syntaxe plus courte. Nous allons voir, ci-dessous, plusieurs façons d'utiliser les conditions inlines de JSX.
 
-### Inline If with Logical && Operator {#inline-if-with-logical--operator}
+### Inline If avec l'opérateur logique && {#inline-if-with-logical--operator}
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Vous pouvez [intégrer des expressions dans JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) en les enveloppant dans des accolades. Cela inclut l'opérateur Javascript logique `&&`. Il peut être pratique pour inclure conditionnellement un élément :
```

Comment:
```suggestion
Vous pouvez [utiliser n’importe quelle expression dans du JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) en les enveloppant dans des accolades. Ça vaut aussi pour l'opérateur logique Javascript `&&`. Il peut être pratique pour inclure conditionnellement un élément :
```

- Attention à retranscrire le *any*, qui insiste sur la versatilité / flexibilité du truc
- Reprise du terme générique « utiliser » pour “embed” ici, par cohérence avec l'intro à JSX et les autres textes sur le sujet
- GG le « envelopper »  pour “to wrap” ; dans ce contexte précis, ça passe très bien 👍 
- Fluidification de la deuxième phrase.